var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -28,8 +28,6 @@ class Cp2k(MakefilePackage, CudaPackage):
     version('develop', branch='master', submodules="True")
```

Comment:
```suggestion
    version('master', branch='master', submodules="True")
```

Reply:
I've discussed this also with other parties. Problem being: for CP2K there is only one - the development - branch, while other projects usually have 2 branches following a git-flow style development model. When comparing it to versions it is therefore actually a development version, rather than a final/release version, which is why I picked `develop` rather than `master` (which for many suggests some sort of release branch).

Reply:
I guess from my perspective, `master` and `develop` tend to be similar for most projects. That is, they either have a single `master` branch and tagged releases, or a single `develop` branch and tagged releases. Very few projects actually have both `master` and `develop` branches, where `master` is more stable than `develop`. I tend to keep the same branch names that the developer chose.

Reply:
ok, done